ui/src/main/resources/edu/wpi/grip/ui/pipeline/OutputSocket.fxml

```diff
@@ -9,7 +9,7 @@
 <?import javafx.scene.layout.HBox?>
 <?import javafx.scene.layout.StackPane?>
 <HBox styleClass="socket"
-      fx:controller="edu.wpi.grip.ui.OperationController"
+      fx:controller="edu.wpi.grip.ui.pipeline.OutputSocketController"
```

Comment:
?

Reply:
`fx:controller` is necessary, and `OperationController` is for controlling the UI element displaying an entire operation. `OutputSocketController` is more appropriate.

This also makes the file in line with `InputSocket.fxml`, which already uses `InputSocketController` as its `fx:controller` attribute